Traceback (most recent call last):
  File "/Users/alexischevalier/Python_projects/MLAB_public/w1d4_answers_part2.py", line 149, in <module>
    train(config_dict)
NameError: name 'config_dict' is not defined